flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/TimeAttributesITCase.scala

```diff
@@ -18,8 +18,8 @@
 
 package org.apache.flink.table.runtime.stream
 
-import java.math.BigDecimal
 import java.lang.{Integer => JInt, Long => JLong}
+import java.math.BigDecimal
```

Comment:
nit: Are we following import orders for Scala as described here for Java: http://flink.apache.org/contribute-code.html#imports?